### PR TITLE
Fixing Bug 1754

### PR DESCRIPTION
**Pull request by dsgouda**
@fearthecowboy PTAL

### CODE REVIEW

File: src/generator/AutoRest.CSharp.Unit.Tests/Bug1754.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    {
+
+        public Bug1754(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        /// <summary>
+        ///     https://github.com/Azure/autorest/issues/1754
+        ///     Verifies autorest succesfully generates code for empty path object
+        /// </summary>
```
Comment (olydis) [inline src/generator/AutoRest.CSharp.Unit.Tests/Bug1754.cs]: Wrong comment.

File: src/generator/AutoRest.CSharp.Unit.Tests/Bug1754.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        ///     on HTTP response objects
+        /// </summary>
+        [Fact]
+        public async Task EnsureHttpMessageAsyncDispose()
+        {
+            // simplified test pattern for unit testing aspects of code generation
+            using (var fileSystem = GenerateCodeForTestFromSpec())
+            {
+                // check for the expected class.
+                Assert.True(fileSystem.FileExists(@"GeneratedCode\TestOperationsExtensions.cs"));
```
Comment (fearthecowboy) [inline src/generator/AutoRest.CSharp.Unit.Tests/Bug1754.cs]: Add the check in here to check the generated text for the `.Dispose()`

**Comment by dsgouda** [comment]
@azuresdkci Test this please

**Comment by olydis** [comment]
@azuresdkci retest this please

**Pull request merged**